packages/connect-query-core/package.json

```
@@ -45,6 +47,8 @@
     "@tanstack/query-core": "5.x"
   },
   "files": [
-    "dist/**"
+    "dist/**",
+    "!*.test.js",
+    "!*.test.d.ts"
```

Do we still need those?

```suggestion
    "dist/**"
```